apigee_m10n.links.task.yml

```diff
@@ -4,10 +4,10 @@ apigee_m10n.settings.rate_plan:
   base_route: apigee_m10n.settings.rate_plan
   weight: -10
 
-entity.package.collection:
-  title: 'List packages'
-  route_name: entity.package.collection
-  base_route: apigee_m10n.settings.package
+entity.product_bundle.collection:
+  title: 'List product bundles'
```

Comment:
Nit: Should we keep this to just `List` like core does?

![Screenshot_7_10_19__12_59_PM](https://user-images.githubusercontent.com/124599/60955783-98372380-a312-11e9-9c30-22f4110ae747.jpg)

Reply:
That's a good suggestion. Thanks.